### PR TITLE
feat: add edgedb

### PR DESCRIPTION
**Pull request by AndreasPB**
# Description

This is the first step to get support for users all the ideas we have from that (watch lists, favorites, already watched, etc)

- Adds EdgeDB insert and select queries for country + genre and generated type-safe python code
- Endpoints added/updated for the new queries
- Splits a some functions up for them to work with both Redis and EdgeDB
- A lot of minor tweaks (justfile, printing errors codes... should have done that long ago :D, ect.)

### CODE REVIEW

File: dbschema/default.esdl [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module default {
+  type Person {
+    required property name -> str;
+    multi link media -> Media;
+  }
+
+  type Media {
+    required property title -> str;
+  }
```
Comment (Primdahl26) [inline dbschema/default.esdl:9]: Save this for when we do media and users

File: backend/cli.py [hunk 2]
```diff
@@ -34,6 +41,17 @@
...
 
+@app.command()
+@coroutine
+async def edgedb_demo():
+    genres = await get_genres(db_client)
+    countries = await get_countries(db_client)
+
+    await insert_genres(db_client, data=json.dumps(fix_genre_ampersand(fetch_genres())))
+
+    print("Genres:", genres, "\nCountries:", countries)
```
Comment (Primdahl26) [inline backend/cli.py:52]: Kill before we merge

**Review by Primdahl26 — APPROVED** [review]

File: backend/cli.py (hunk 2 above)
Comment (AndreasPB) [reply backend/cli.py]: 💀

File: dbschema/default.esdl (hunk 1 above)
Comment (AndreasPB) [reply dbschema/default.esdl]: Da

**Pull request merged**